dom-element.js

```diff
@@ -93,25 +93,43 @@ DOMElement.prototype.insertBefore =
         return elem
     }
 
-DOMElement.prototype.setAttribute =
-    function _Element_setAttribute(name, value) {
-        this._attributes[name] = value
+DOMElement.prototype.setAttributeNS =
+    function _Element_setAttributeNS(namespace, name, value) {
+        var localName = name.substr(name.indexOf(":") + 1)
```

Comment:
`indexOf` conveniently returns `-1` if the name doesn't contain a : so it still works correctly, but you're right that code is confusing. I will push an update to make it more explicit.